com.microsoft.mrtk.graphicstools.unity/Runtime/Utilities/FlyCameraController.cs

```diff
@@ -70,7 +71,7 @@ public void UpdateTransform(Transform t)
         public bool InvertY = false;
 
         [Header("Other Settings")]
-        public bool showControlsText = true;
+        public bool showControlsText = false;

```

Comment:
Let's not show UI initially so people don't pay the cost of UIGUI unintentionally.